### ISSUE TITLE
Set terms of service for BioSimulations

### ISSUE THREAD

**Issue by jonrkarr** [biosimulations/Biosimulations#573]
@moraru @mpw6, what terms of service do you want to use for BioSimulations?

**Comment by jonrkarr** [comment]
We also need to pick a physical address.

**Comment by bilalshaikh42** [comment]
We also need a privacy policy, especially regarding Eu and California. 
I think Uniprot provides a good example
https://www.uniprot.org/help/privacy

**Comment by jonrkarr** [comment]
The EBI has a similar policy for BioModels:
https://www.ebi.ac.uk/data-protection/privacy-notice/embl-ebi-public-website

We also need a disclaimer. Example from BioModels:
http://www.ebi.ac.uk/biomodels/termsofuse

**Comment by bilalshaikh42** [comment]
https://www.omg.org/cloud/deliverables/CSCC-Practical-Guide-to-Cloud-Service-Agreements.pdf

**Comment by bilalshaikh42** [comment]
Another good example: 
https://orcid.org/content/orcid-public-client-terms-service

**Comment by jonrkarr** [comment]
I drafted terms of service and privacy policies, mainly following the examples of BioModels and the EBI. UniProt's policy (also EBI) is similar. ORCID's policies are much more formal.

For a physical address, I used UConn since this is where the computations are actually being run and the data is being stored.

I'm closing this. This, and all other content will need to be reviewed and proofread prior to publication.
